web/packages/test/scripts/build-binary.sh

```diff
@@ -14,8 +14,8 @@ build_binaries() {
     check_local_changes "polkadot"
     check_local_changes "substrate"
 
-  # Check that all 3 binaries are available and no changes made in the polkadot and substrate dirs
-    if [[ ! -e "target/release/polkadot" || ! -e "target/release/polkadot-execute-worker" || ! -e "target/release/polkadot-prepare-worker" || "$changes_detected" -eq 1 ]]; then
```

Comment:
Remove the change detect for building the polkadot binary, IMO in most cases we only change the cumulus codes, and building polkadot is unnecessary.  In case of a major substrate version upgrade, we can build Polkadot manually.